Fix invalid usage of CHECK_INCLUDE_FILES that causes:

CMake Error at /opt/local/share/cmake-3.24/Modules/CheckIncludeFiles.cmake:70 (message):
  Unknown arguments:

    HAVE_MATH_H

https://trac.openmodelica.org/OpenModelica/attachment/ticket/5409/paradiseo-cmake.patch
--- ConfigureChecks.cmake.orig	2023-09-01 18:37:51.000000000 -0500
+++ ConfigureChecks.cmake	2023-09-01 18:37:51.000000000 -0500
@@ -8,15 +8,15 @@
 
 CHECK_LIBRARY_EXISTS(m	cos	"/usr/lib"	HAVE_LIBM)
 
-CHECK_INCLUDE_FILES(math.h	"math.h"	HAVE_MATH_H)
-CHECK_INCLUDE_FILES(stdio.h	"stdio.h"	HAVE_STDIO_H)
-CHECK_INCLUDE_FILES(stdlib.h	"stdlib.h"	HAVE_STDLIB_H)
-CHECK_INCLUDE_FILES(string.h	"string.h"	HAVE_STRING_H)
-CHECK_INCLUDE_FILES(strings.h	"strings.h"	HAVE_STRINGS_H)
-CHECK_INCLUDE_FILES(malloc.h	"malloc.h"	HAVE_MALLOC_H)
-CHECK_INCLUDE_FILES(limits.h	"limits.h"	HAVE_LIMITS_H)
-CHECK_INCLUDE_FILES(unistd.h	"unistd.h"	HAVE_UNISTD_H)
-CHECK_INCLUDE_FILES(stdint.h	"stdint.h"	HAVE_STDINT_H)
+CHECK_INCLUDE_FILES("math.h"	HAVE_MATH_H)
+CHECK_INCLUDE_FILES("stdio.h"	HAVE_STDIO_H)
+CHECK_INCLUDE_FILES("stdlib.h"	HAVE_STDLIB_H)
+CHECK_INCLUDE_FILES("string.h"	HAVE_STRING_H)
+CHECK_INCLUDE_FILES("strings.h"	HAVE_STRINGS_H)
+CHECK_INCLUDE_FILES("malloc.h"	HAVE_MALLOC_H)
+CHECK_INCLUDE_FILES("limits.h"	HAVE_LIMITS_H)
+CHECK_INCLUDE_FILES("unistd.h"	HAVE_UNISTD_H)
+CHECK_INCLUDE_FILES("stdint.h"	HAVE_STDINT_H)
 
 
 # Use check_symbol_exists to check for symbols in a reliable
